src/router.js

```diff
@@ -15,6 +15,11 @@ export default new Router({
       path: '/live',
       name: 'live',
       component: () => import('./views/Live.vue')
-    }
+    },
+    {
+      path: '/myteams',
+      name: 'MyTeams',

```

Comment:
Please don't capitalize here. All code styles should be the same format.

Reply:
thanks for your suggestion.